client/src/components/signUpForm/PhysicalCharacteristics/index.js

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import { Typography, Hidden, CardMedia } from '@material-ui/core';
+
+import InputField from '../../InputField';
+import Container from '../../Container';
+import ButtonComponent from '../../Button';
+import SelectComponent from '../../Select';
+import {
+  activitySchema,
+  ageSchema,
+  goalWeightSchema,
+  heightSchema,
+  weightSchema,
+} from './validation';
+
+const { shape, func } = PropTypes;
+
+const useStyle = makeStyles((theme) => ({
+  input: {
+    color: theme.customColors[1],
+    width: '80%',
+    backgroundColor: theme.customColors[7],
+  },
+}));
+
+function PhysicalCharacteristics(props) {
+  const {
+    methods: { handleBack, setData },
+  } = props;
+  const classes = useStyle();
+
+  const [age, setAge] = useState('');
+  const [activity, setActivity] = useState(0);
+  const [weight, setWeight] = useState('');
+  const [height, setHeight] = useState('');
+  const [goalWeight, setGoalWeight] = useState('');
+
+  const [ageError, setAgeError] = useState(false);
+  const [activityError, setActivityError] = useState(false);
+
+  const [weightError, setWeightError] = useState(false);
+  const [heightError, setHeightError] = useState(false);
+  const [goalWeightError, setGoalWeightError] = useState(false);
+
+  return (
+    <Container screenSize="lg" direction="row" itemColumns="6" spacing={2}>
+      <Hidden smDown key="1">
+        <CardMedia
+          component="img"
+          height="600"
+          image="./signupImages/3.png"
+          title="main"
+        />
+      </Hidden>
+      <Container key="2" direction="column" itemColumns="12" spacing={5}>
+        <Typography key="1" variant="h5">
+          Select Your Physical Characteristics :-
+        </Typography>
+        <form key="22" style={{ width: '100%' }}>
+          <Container direction="column" itemColumns="12" spacing={5}>
+            <SelectComponent
+              key="1"
+              value={activity}
+              handleChange={async ({ target: { value } }) => {
+                setActivity(value);
+                const isValid = await activitySchema.isValid({ activity });
+                setActivityError(!isValid);
```

Comment:
My comment about duplication inside the change handlers from `BasicUserInfo` above could also help here.